examples/advection_1d/test_advection_nonunif.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import absolute_import
+def test_1d_advection():
+    """test_1d_advection
+
+    tests against expected classic, sharpclaw, and high-order weno results """
+
+    import advection_1d
```

Comment:
Do you need this import?

Reply:
not anymore, I had it when I only tested classic for nonuniform example and tested sharpclaw for standard example.